packages/svelte/src/SupaAuthHelper.svelte

```diff
@@ -9,8 +9,9 @@
 
   // Props
   export let supabaseClient: SupabaseClient;
-  export let callbackUrl = '/api/auth/callback';
-  export let profileUrl = '/api/auth/user';
+  export let endpointPrefix = '/api/auth';
```

Comment:
It's probably worth pulling thru the `ENDPOINT_PREFIX` const from the `@supabase/auth-helpers-shared` package too.